compute/disks/create_hyperdisk.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "cloud.google.com/go/compute/apiv1/computepb"
+	"google.golang.org/protobuf/proto"
+)
+
+// [START compute_hyperdisk_create]
+// createHyperdisk creates a new Hyperdisk in the specified project and zone.
+func createHyperdisk(w io.Writer, projectId, zone, diskName string, diskSizeGb int64) error {
```

Comment:
why did you choose to include disk size here? seems like an unnecessary parameter to show usage.
the comment below includes optional fields that your function does not accept. please delete those.